### DISCUSSION TITLE
Verification e-mail is not send, even after 30 minutes of waiting

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#23610] [New to GitHub]
<p>I want to add a new e-mail to my account to use in the future as my main one, but no matter how long I wait, the verification e-mail just won’t arrive in my mailbox.</p>
<p>I so far waited over 30 minutes for it to arrive and while I received a mail informing me about the mail being added to my account, is the other one still not here.<br>
I did check that the e-mail address is written correctly and I also tested that the e-mail is properly forwarded to my account, which it is.</p>
<p>While I doubt that the current <a href="https://www.githubstatus.com/incidents/gh0vvxtlj5v7" rel="noopener nofollow ugc">GitHub Actions incident</a> could be a possible cause, do I still wonder if it actually could be a cause for this heavy delay…</p>
<p>All I want is to have my e-mail verified but this issues slows me down more than it should…</p>
<p>Is there any way to “debug” this? Like to find out if GitHub actually is able to send it and if they did so?</p>

**Comment by Andre601** [comment]
<p>Quick update: After a long, long wait, did the verification E-Mail finally arrive, but when I click the verify button do I get an error…</p>

**Comment by jfullstackdev** [comment]
<p>yup, let’s see, the problem is still on-going although it’s GitHub Actions, that might affect other services, and hopefully it will be okay as soon as possible,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/7/a7e771532dd328aef9bce3074cb314aabcb29270.png" data-download-href="/uploads/short-url/nXlBh3eVA0o0LvtBfcN4YAL2yTS.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181078214-eba76bde-219a-4dea-a6ea-b29ad01181c5.png" alt="image" data-base62-sha1="nXlBh3eVA0o0LvtBfcN4YAL2yTS" width="690" height="128" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/7/a7e771532dd328aef9bce3074cb314aabcb29270_2_690x128.png, https://aws1.discourse-cdn.com/github/optimized/3X/a/7/a7e771532dd328aef9bce3074cb314aabcb29270_2_1035x192.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/a/7/a7e771532dd328aef9bce3074cb314aabcb29270_2_1380x256.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/7/a7e771532dd328aef9bce3074cb314aabcb29270_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1540×287 42.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p><a href="https://www.githubstatus.com/" class="onebox" target="_blank" rel="noopener nofollow ugc">https://www.githubstatus.com/</a></p>

**Reply by vincentgpower** [reply]
same issue here last 2 hours

**Reply by itpofy2024o** [reply]
degenerated github on ai ascension

**Reply by zatokar1** [reply]
same issue 4 years later... fck this feature worked 15 years ago, we live in AI times we email verification still doesnt work..

**Reply by itpofy2024o** [reply]
well this is a business let do it

**Comment by Andre601** [comment, marked as answer]
<p>I did get a new verification e-mail and this time the URL did work, so the issue has been resolved for me… Tho I now encounter another issue, but I will make a separate post for that one, since it’s not directly related to this… I think.</p>

**Comment by ErnistS** [comment]
<p>Hi I am not getting the verification email and I have waiter for 6 hours please help, as now it’s some how not allowing me to log into my Hotmail account as it’s just access to the auth app and I can’t recover it</p>

**Comment by jfullstackdev** [comment]
<aside class="quote no-group" data-username="ErnistS" data-post="5" data-topic="219785">
<div class="title">
<div class="quote-controls"></div>
 ErnistS:</div>
<blockquote>
<p>not allowing me to log into my Hotmail account as it’s just access to the auth app and I can’t recover it</p>
</blockquote>
</aside>
<p>you mean you cannot log in at your email? then you cannot verify it,</p>
<p>you can still add the email but not verified to get all the past commits,</p>

**Comment by ErnistS** [comment]
<p>Yes it eventually came through and I clicked the link then it all froze and my auth app then I could not access it. And now I can’t access my email account or auth app it wiped the backup from my cloud</p>
<p>I am not sure what to do as hot ail can’t do anything to help recover the account</p>

**Comment by jfullstackdev** [comment]
<p>do you have a backup email? you can use that to retrieve the other email, or a mobile phone to send verification code to retrieve the email</p>

**Reply by ErnistS** [reply]
I had it working then it stopped again

**Comment by Brunostar** [comment]
Hi, I have been waiting for 12 hours and have not received my verification email. From the discussion above, It seems like the email will come with time, but it will be of no use, since the verification link would probably have expired.

**Comment by daresut** [comment]
Hi, I have been experiencing strange delays with verification emails. It takes an hour or more. This problem appears to be recurrent with github since at least 2017.

**Reply by Brunostar** [reply]
Hi it took more than 24 hours for my verification email to arrive. The link was still functional, and I successfully verified my email. So my suggestion: be patient.

**Comment by nk9** [comment]
Waited about 30 minutes for mine. Pro tip: don't request a confirmation code multiple times. The first one will be invalidated, so this will just delay the arrival of a useable code. >_<

**Comment by obia5313** [comment]
чт, 4 мая 2023 г., 3:21 AM Nick Kocharhook ***@***.***>:

**Comment by SaketGudimella** [comment]
I'm not able to add a new email address to my github account

**Comment by RichardScottOZ** [comment]
Not getting them at all for some email but am for others, quite strange - when all the email addresses are fine.

**Comment by nadirmusija** [comment]
Mine is not coming at all, even though i wait

**Comment by Oudwins** [comment]
I am also having this issue atm.... It just doesn't come, ever

**Comment by olaobey** [comment]
What could have happened I used localhost to test signup and it works and email verification code works but when I use base url it taks long time before the sign up work and email verification does not work pls help me

**Comment by scottenders** [comment]
Same issue, I am unable to verify any new email addresses I try to add

**Reply by scottenders** [reply]
Found my issue was the mail server was blocking on my behalf. Once that was removed, I was able to verify

**Comment by jneubert** [comment]
Same for me - tried some weeks ago, and today again, for two different email addresses. Neither did work.

**Comment by Andre601** [comment]
I'll mark this as closed now because A) it's quite old (I made this in 2021) and B) it's not meant as a general discussion to share your "I have that issue too" comments honestly...

If you have issues, make your own discussion or contact the GitHub Support... But posting here brings nothing useful to the table.

Personally would I apreciate if no further replies are given or if staff could lock this discussion, as there is no point in extra comments being made to necropost here.

**Comment by jeffabenoja** [comment]
This is same to me I tried to login my google account even try a new account and required for email verification. But ain't receiving any verification code from my email.

**Comment by buni-gutz** [comment]
hey im trying to do this right now and having issues

**Comment by Rokata9** [comment]
At the moment I also can't login on my new machine no matter how many times I click re-send, is there some outage or degraded experience?
